modules/apps/frontend-data-set/frontend-data-set-builder-web/src/main/java/com/liferay/frontend/data/set/builder/web/internal/portlet/FDSBuilderPortlet.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.data.set.builder.web.internal.portlet;
+
+import com.liferay.frontend.data.set.builder.web.internal.constants.FDSBuilderPortletKeys;
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCPortlet;
+
+import javax.portlet.Portlet;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Marko Cikos
+ */
+@Component(
+	property = {
+		"com.liferay.portlet.display-category=category.hidden",
+		"com.liferay.portlet.layout-cacheable=true",
+		"javax.portlet.display-name=Frontend Data Set Builder",
```

Comment:
Yes, it will not be available to be dropped. I believe that URLs are based on `javax.portlet.name` property. Besides the drop selection, display name would appear on a normal page portlet, for admins, when you hover and go to configuration. But, for a global menu app, it never shows. I don't think end user will be able to view this in any context.

I just checked and the property is not mandatory, so we can just remove it. It should be safe.